kp_test.go

```diff
@@ -278,6 +286,21 @@ func TestKeys(t *testing.T) {
 				return nil
 			},
 		},
+		{
+
+			"Wrap V2 With KeyVersionID",
+			func(t *testing.T, api *API, ctx context.Context) error {
+				MockAuthURL(keyURL, http.StatusOK, keyActionWithKeyVersion)
+				MockAuthURL(keyURL, http.StatusOK, keyActionWithKeyVersion)
+
+				wrapWithKeyVersion, err := api.WrapV2(ctx, testKey, []byte(keysActionPT.PlainText), nil)
+				assert.NoError(t, err)
+				assert.NotEmpty(t, wrapWithKeyVersion)
+				assert.NotEqual(t, wrapWithKeyVersion.PlainText, wrapWithKeyVersion.CipherText)
+
```

Comment:
pls, validate the key version as well.
```suggestion
                        assert.NotEmpty(t, wrapWithKeyVersion.KeyVersion.ID)
```

Reply:
I would suggest to verify the value of version id here:
```suggestion
				assert.Equal(t, testKey, wrapWithKeyVersion.KeyVersion.ID)
```